account_consolidation/wizard/consolidation_check_view.xml

```diff
@@ -1,20 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" ?>
```

Comment:
We should keep the encoding for XML files.

Reply:
in v12 no need to keep them anymore, you even will have lint complaining on it

Reply:
Can you point to an example of lint complaining ? In odoo, most of the XML files still have the encoding set...

Reply:
I will turn it back, can't find, but definitely  saw somewhere :(